components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -1020,24 +1023,24 @@
 <!-- Deep convection scheme -->
 <deep_scheme>ZM</deep_scheme>
 <deep_scheme clubb_do_deep="1" >CLUBB_SGS</deep_scheme>
+<deep_scheme use_MMF="1"       >off      </deep_scheme>
 
 <!-- PBL scheme -->
 <eddy_scheme pbl="uw"             >diag_TKE </eddy_scheme>
 <eddy_scheme pbl="hb"             >HB       </eddy_scheme>
 <eddy_scheme pbl="hbr"            >HBR      </eddy_scheme>
 <eddy_scheme pbl="clubb_sgs"      >CLUBB_SGS</eddy_scheme>
+<eddy_scheme use_MMF="1"          >off      </eddy_scheme>
```

Comment:
I thought MMF had to use the HB pbl scheme now? Is this something different?

Reply:
it does... but it has it's own version... I thought about adding a special option to make this clear, but wasn't sure if it was necessary. The PBL scheme calculations are necessary (unfortunately), but they are only ever called from the code in the physics/crm directory, and I don't foresee any situation where we want to have multiple PBL options.